db/seeds_for_general_election.rb

```diff
@@ -24,7 +24,7 @@
 
 parties_for_ge.each do |party_attributes|
   party = Party.find_or_initialize_by(name: party_attributes[:name])
-  party.update!(party_attributes.slice(:smv_code, :color))
+  party.update!(party_attributes.slice(:smv_code, :color, :shortName))
```

Comment:
I **think** this is the right thing to do? I don't know anything about how Ruby does databases.

Reply:
It's not as simple as that to add it to the database. What about just keeping it in the model ?